models/DataObject/ClassDefinition/Data/Table.php

```diff
@@ -497,38 +497,38 @@ public function checkValidity($data, $omitMandatoryCheck = false)
      * @param DataObject\AbstractObject $object
      * @param array $params
      *
-     * @return string
+     * @return string|null
      */
     public function getForCsvExport($object, $params = [])
     {
         $data = $this->getDataFromObjectParam($object, $params);
         if (is_array($data)) {
             return base64_encode(Serialize::serialize($data));
-        } else {
-            return null;
         }
+
+        return null;
     }
 
     /**
      * @param string $importValue
      * @param null|DataObject\Concrete $object
-     * @param mixed $params
+     * @param array $params
      *
-     * @return mixed|null
+     * @return array|null
      */
     public function getFromCsvImport($importValue, $object = null, $params = [])
     {
         $value = Serialize::unserialize(base64_decode($importValue));
         if (is_array($value)) {
             return $value;
-        } else {
-            return null;
         }
+
+        return null;
     }
 
     /**
      * @param DataObject\Concrete|DataObject\Objectbrick\Data\AbstractData|DataObject\Fieldcollection\Data\AbstractData $object
```

Comment:
wondering if Localizedfields are missing here

Reply:
I add it